attest/verifier/types/src/verification.rs

```diff
@@ -44,6 +46,9 @@ impl Message for DcapEvidence {
         let collateral_bytes: Vec<u8> =
             mc_util_serial::serialize(&self.collateral).expect("Failed to serialize Collateral");
         encoding::bytes::encode(TAG_DCAP_EVIDENCE_COLLATERAL, &collateral_bytes, buf);
+        let report_data_bytes: Vec<u8> = mc_util_serial::serialize(&self.report_data)
+            .expect("Failed to serialize EnclaveReportDataContents");
+        encoding::bytes::encode(TAG_DCAP_EVIDENCE_REPORT_DATA, &report_data_bytes, buf);
```

Comment:
I think this will likely be the last field added to the DcapEvidence.
While I'm a fan of the 3 strike rule:
1. it needs to be there.
2. it might be duplicate logic but ok.
3. your out and should probably dry up the code.

I don't think trying to macroizing this for 3 fields will gain us much and will likely make it harder to understand what's going on.
At the same time I'm not opposed to it if we feel it's worth it.

Reply:
I think macroizing this is unnecessary and will just make reading it harder.